### ISSUE TITLE
the loss is too large

### ISSUE THREAD

**Issue by buptlj** [Time0o/pytorch-colorful-colorization#40]
I train the model from scratch and use the default parameters，but the loss remains too large(1e4) after 40000 iters.

**Comment by SUNJIMENG** [comment]
Excuse me, I meet the same issue with you, have you solved it now, please?

**Comment by duxiangcheng** [comment]
I meet the same problem, have you solved it now?

**Comment by Time0o** [comment]
I would love to help you with this, but you need to provide more information, i.e. on which images are you training and with on what OS/with which GPU, Pytorch version etc.
